src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -1473,7 +1473,6 @@ If such a class is used as a base class and if the deriving class defines a dest
     <value>Predefined type '{0}' is not defined or imported</value>
   </data>
   <data name="ERR_PredefinedValueTupleTypeNotFound" xml:space="preserve">
-    <!-- We need a specific error code for ValueTuple as an IDE codefix depends on it (AddNuget) -->
     <value>Predefined type '{0}' is not defined or imported</value>
```

Comment:
It is strange that the comment is gone..

Reply:
perhaps edit the XML directly?

---
In reply to: [172299725](https://github.com/dotnet/roslyn/pull/25186#discussion_r172299725) [](ancestors = 172299725)